api/infrastructure/parallelcluster-api.yaml

```diff
@@ -725,6 +725,8 @@ Resources:
               - logs:PutRetentionPolicy
               - logs:DescribeLogGroups
               - logs:CreateLogGroup
+              - logs:TagLogGroup
```

Comment:
TagLogGroup is on deprecation path see https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_TagLogGroup.html#:~:text=The%20TagLogGroup%20operation%20is%20on%20the%20path%20to%20deprecation.%20We%20recommend%20that%20you%20use%20TagResource%20instead.

Reply:
Nice catch. Done